.ceramicDev.config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "anchor": { },
+  "http-api": {
+    "cors-allowed-origins": [
+      ".*"
+    ],
+    "admin-dids": [
+      "did:key:z6MktbKJrMnhVJ37QFTo12911ycm2juKDUzWHDVETu9s5a9T"
```

Comment:
This requires a shared dev seed in `desci-codex` which I'm not sure how to share, same as what'll go in the frontend `.env` before proper wallet integration :thinking: